components/03-sections/06-blockquote/blockquote--image-heading.hbs

```diff
@@ -13,7 +13,7 @@
                     </div>
                 </div>
                 <div class="col-xxl-8 col-xl-8 col-lg-8 col-md-8 col-sm-12">
-                    <div class="image_right-cntnt">
```

Comment:
should we standardize this .image-right-content instead of using _ while we have the chance?

Reply:
Yes. Will do in next PR on this